dogs-initial/src/main/java/com/lambdaschool/dogsinitial/model/MessageDetail.java

```diff
@@ -0,0 +1,57 @@
+package com.lambdaschool.dogsinitial.model;
+
+import java.io.Serializable;
+
+public class MessageDetail implements Serializable
+{
+    private String text;
+    private int priority;
+    private boolean secret;
+
```

Comment:
Keep in mind that while the `priority` and `secret` variables can be useful for your `MessageDetail` object, they are not needed for this application.